migrations/versions/c02a75d4f418_.py

```diff
@@ -0,0 +1,160 @@
+"""empty message
```

Comment:
License.

Reply:
We don't normally put license headers on these autogenerated migration files (check all deleted migration files below/above).